jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/JmeterDsl.java

```diff
@@ -188,7 +188,32 @@ public static DslThreadGroup threadGroup(String name) {
    * @since 0.33
    */
   public static DslSetupThreadGroup setupThreadGroup(ThreadGroupChild... children) {
-    return new DslSetupThreadGroup(Arrays.asList(children));
+    return new DslSetupThreadGroup(null, Arrays.asList(children));
+  }
+
+  /**
+   * This allowing to set a name on the setup thread group.
+   * <p>
+   * Setting a proper name allows to properly identify the requests generated in each thread group.
+   *
+   * @see DslSetupThreadGroup
+   * @since 0.33.2
+   */
+  public static DslSetupThreadGroup setupThreadGroup(String name, ThreadGroupChild... children) {
+    return new DslSetupThreadGroup(name, Arrays.asList(children));
+  }
+
+  /**
+   * This allowing to create empty the setup thread group,
```

Comment:
The wording looks off, aside from similar notes in previous comment. We can fix this after merge if you are ok with it.